parser/parser.go

```diff
@@ -25,39 +25,50 @@ import (
-	toowner           = transaction.Flag("toowner", "New owner of the UTXO").Required().String()
-	fromowner         = transaction.Flag("fromowner", "from an owner of the UTXO").Required().String()
-	privatekey        = transaction.Flag("privatekey", "privatekey to sign from owner of original UTXO").Required().String()
-	toamount          = transaction.Flag("toamount", "Amount to transact").Required().Uint()
-	fromutxo          = transaction.Flag("fromutxo", "utxo position to send from").Required().Uint()
-	watcherSubmitURL  = transaction.Flag("watcher", "FQDN of the Watcher in the format http://watcher.path.net").Required().String()
-	exit              = kingpin.Command("exit", "Standard exit a UTXO back to the root chain.")
-	watcherExitURL    = exit.Flag("watcher", "FQDN of the Watcher in the format http://watcher.path.net").Required().String()
-	contractExit      = exit.Flag("contract", "Address of the Plasma MoreVP smart contract").Required().String()
-	utxoPosition      = exit.Flag("utxo", "UTXO Position that will be exited").Required().String()
-	exitPrivateKey    = exit.Flag("privatekey", "Private key of the UTXO owner").Required().String()
-	clientExit        = exit.Flag("client", "Address of the Ethereum client. Infura and local node supported https://rinkeby.infura.io/v3/api_key or http://localhost:8545").Required().String()
+	get              = kingpin.Command("get", "Get a resource.").Default()
+	getUTXO          = get.Command("utxos", "Retrieve UTXO data from the Watcher service")
+	watcherURL       = get.Flag("watcher", "FQDN of the Watcher in the format http://watcher.path.net").Required().String()
+	ownerUTXOAddress = get.Flag("address", "Owner address to search UTXOs").String()
+	getBalance       = get.Command("balance", "Retrieve balance of an address from the Watcher service")
+	status           = get.Command("status", "Get status from the Watcher")
+
+	deposit         = kingpin.Command("deposit", "Deposit ETH or ERC20 into the Plamsa MoreVP smart contract.")
+	privateKey      = deposit.Flag("privatekey", "Private key of the account used to send funds into Plasma MoreVP").Required().String()
+	client          = deposit.Flag("client", "Address of the Ethereum client. Infura and local node supported https://rinkeby.infura.io/v3/api_key or http://localhost:8545").Required().String()
+	contract        = deposit.Flag("contract", "Address of the Plasma MoreVP smart contract").Required().String()
+	depositOwner    = deposit.Flag("owner", "Owner of the UTXOs").Required().String()
+	depositAmount   = deposit.Flag("amount", "Amount to deposit in wei").Required().Uint64()
+	depositCurrency = deposit.Flag("currency", "Currency of the deposit. Example: ETH").Required().String()
+
+	transaction      = kingpin.Command("send", "Create a transaction on the OmiseGO Plasma MoreVP network")
+	toowner          = transaction.Flag("toowner", "New owner of the UTXO").Required().String()
```

Comment:
As `transaction` has been renamed to `send`, it would make sense to change to these variables to reflect that.

Reply:
good point !